iotlabcli/parser/common.py

```diff
@@ -128,6 +128,10 @@ def catch_missing_auth_cli():
 def main_cli(function, parser, args=None):  # flake8: noqa
     """ Main command-line execution. """
     args = args or sys.argv[1:]
+    if len(args) == 0:
```

Comment:
upon your request, I looked "around there" and proposed instead to add the test here: https://github.com/iot-lab/cli-tools/blob/show_usage_if_no_args/iotlabcli/integration/test_cli_tools_help.py#L20

whiche slightly extended into the file as it is now.